src/thingset_txt.c

```diff
@@ -278,6 +278,14 @@ static int txt_serialize_path(struct thingset_context *ts,
     return -THINGSET_ERR_INTERNAL_SERVER_ERR;
 }
 
+static int txt_serialize_metadata(struct thingset_context *ts,
+                                  const struct thingset_data_object *object)
+{
+    /* not used for text mode */
```

Comment:
While serializing the path does not make sense for txt mode, the metadata could also be exposed in the text mode in the future, so I suggest a different comment:

```suggestion
    /* not yet implemented for text mode */
```